modules/build/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -346,9 +345,31 @@ final case class BuildOptions(
     (sv, sbv)
   }
 
+  def computeScalaThreeNightlyVersions(): Either[BuildException, (String, String)] = either {
+    import coursier.{Module, Versions}
```

Comment:
NIT: clean up imports and I would prefer to have them on the top of the file  (maybe excluding ExecutionContext one)

Reply:
Thank you I would do so